js/common.js

```diff
@@ -1094,13 +1094,21 @@ var App = new function() {
         this.currentNoteUnsaveable = function() {
             return currentNote.isMissingResourceData();
         };
+        
+        this.showDoneButton = function(){
+             elSave.classList.add(CLASS_WHEN_VISIBLE);
+         }; 
+         
+         this.hideDoneButton = function(){
+             elSave.classList.remove(CLASS_WHEN_VISIBLE);
+         };
```

Comment:
indentation between lines 1099 and 1104 is wrong; they should be aligned with the `this.currentNoteUnsaveable` (right now there is an extra space at the beginning of the lines)